repository.json

```diff
@@ -678,6 +678,34 @@
             "version": "1.9.7",
             "url": "https://bitbucket.org/ariya/phantomjs/downloads/phantomjs-1.9.7-windows.zip",
             "fileMatchInside": ".*/phantomjs.exe$"
+        },
+        {
+            "name": "marionette",
+            "platform": "windows",
+            "bit": "32",
+            "version": "0.8.0",
+            "url": "https://github.com/mozilla/geckodriver/releases/download/v0.8.0/geckodriver-v0.8.0-win32.zip"
+        },
+        {
+            "name": "marionette",
+            "platform": "linux",
+            "bit": "64",
+            "version": "0.8.0",
+            "url": "https://github.com/mozilla/geckodriver/releases/download/v0.8.0/geckodriver-0.8.0-linux64.gz"
+        },
+        {
+            "name": "marionette",
+            "platform": "mac",
+            "bit": "32",
+            "version": "0.8.0",
+            "url": "https://github.com/mozilla/geckodriver/releases/download/v0.8.0/geckodriver-0.8.0-OSX.gz"
+        },
+        {
+            "name": "marionette",
+            "platform": "mac",
+            "bit": "64",
+            "version": "0.8.0",
+            "url": "https://github.com/mozilla/geckodriver/releases/download/v0.8.0/geckodriver-0.8.0-OSX.gz"
         }
```

Comment:
The problem with that fallback solution is that it would not work when trying to detect bits for Windows 10 and the Internet Explorer driver. 

The alternatives for me seems to be to
- Remove the bit detection from the plugin and enforce the user to set it.
- Make the bit detection simple and always download the bit version of the OS 
- Somehow make it configurable through the repository.json

But that's another PR as you say.

I'll merge this PR and add the if case myself and then do a release ASAP.